js/core/form/editors/AvatarEditorView.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Developer: Oleg Verevkin
+ * Date: 02/19/2016
+ * Copyright: 2009-2016 Comindware®
+ *       All Rights Reserved
+ * Published under the MIT license
+ */
+
+'use strict';
+
+import {helpers} from '../../utils/utilsApi';
+import BaseItemEditorView from './base/BaseItemEditorView';
+import template from './templates/avatarEditor.hbs';
+
+const defaultOptions = {
+    autoUpload: false,
+    refreshPreviewAfterUpload: false
```

Comment:
Instead, this option should be false in most cases because we have already got in prevew what we're going to upload, so we don't need to refresh it (unless another image is returned from server). I suggest to preserve it as is.